jdaviz/core/helpers.py

```diff
@@ -982,3 +992,85 @@ def _next_subset_num(label_prefix, subset_groups):
                         max_i = i
 
     return max_i + 1
+
+
+class CubeConfigHelper(ImageConfigHelper):
+    """Base config helper class for cubes"""
+
+    _loaded_flux_cube = None
+    _loaded_uncert_cube = None
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def select_slice(self, value):
+        """
+        Select the slice closest to the provided value.
+
+        Parameters
+        ----------
+        value : float or int, optional
+            Slice value to select in units of the x-axis of the profile viewer.
+            The nearest slice will be selected if "snap to slice" is enabled in
+            the slice plugin.
+        """
+        msg = SliceSelectSliceMessage(value=value, sender=self)
+        self.app.hub.broadcast(msg)
+
+
+# TODO: We can remove this when specutils supports it, i.e.,
+#       https://github.com/astropy/specutils/issues/592 and
+#       https://github.com/astropy/specutils/pull/1009
+# NOTE: Cannot use custom_write decorator from specutils because
+#       that involves asking user to manually put something in
+#       their ~/.specutils directory.
+
+def jdaviz_cube_fitswriter(spectrum, file_name, **kwargs):
```

Comment:
Why is this here? It should not exist anymore. Rebased gone bad?

Reply:
I think you're right, thanks for this catch, @pllim! Just pushed a commit to remove this.